src/stories/textarea/fd-textarea-stories.ts

```diff
@@ -0,0 +1,44 @@
+import { moduleMetadata } from '@storybook/angular';
+import { withKnobs, boolean, select, text } from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
+import { ReactiveFormsModule } from '@angular/forms';
+
+import { FormControlDirective, FormModule } from 'libs/core/src/lib/form/public_api';
+
+export default {
+    title: 'Fd text-area',
+    component: FormControlDirective,
+    moduleMetadata: moduleMetadata,
+    decorators: [
+        withKnobs,
+        withA11y,
+        moduleMetadata({
+            imports: [FormModule, ReactiveFormsModule],
+            declarations: []
+        })
+    ]
+};
+
+export const TextArea = () => ({
+    template:
+        `
+        <div fd-form-item>
+        <label fd-form-label for="input-1">Default Text Area</label>
+        <textarea fd-form-control type="text" id="input-1" placeholder="Field placeholder text"
```

Comment:
Based on this PR https://github.com/SAP/fundamental-ngx/pull/2401/files the values here should be dynamic (knobs)

Reply:
> the filename is incorrect. it should be `fd-textarea.stories.ts`.
> I am not sure how you ran this.
> 
Weird it was working mustve clicked it by accident before pushing
> * [ ]  can you add a knob for placeholder, or at least set one

Sure! thought we were only testing the API on the website but will add that now!
> * [ ]  It would be good to have an example with form message or maybe create another story for textarea with form message

Each storybook PR was meant to do one single thing. The storybook with form-messages will be in another PR!
> * [ ]  option to disable the textarea
Added